### ISSUE TITLE
How to save checkpoint files?

### ISSUE THREAD

**Issue by NDTanja** [zalandoresearch/flair#959]
I started to train a language model and set the checkpoint parameter to true. But unfortunately, the script does not write out checkpoint.pt model files at the end of each epoch.
My script looks like this:

```

model: LanguageModel = FlairEmbeddings('de-forward').lm

is_forward_lm = True

dictionary: Dictionary = Dictionary.load('chars')

corpus = TextCorpus('//home//flair_embedding_input//corpus',
                    dictionary,
                    is_forward_lm,
                    character_level=True)

language_model = LanguageModel(dictionary,
                               is_forward_lm,
                               hidden_size=1024,
                               nlayers=1)

trainer = LanguageModelTrainer(language_model, corpus)

trainer.train('resources/taggers/language_model',
              sequence_length=250,
              mini_batch_size=100,
              learning_rate=20,
              anneal_factor=0.4,
              patience=25,
              checkpoint=True)

language_model.save('flair_embedding_modell_2.pt')
```

**Comment by alanakbik** [comment]
Hello @NDTanja it will only save a model for each full epoch, i.e. for each iteration over all training splits. So it won't checkpoint after each split, but rather when it has gone through all splits once.

**Comment by NDTanja** [comment]
Thank you for your quick response! But I did not split my training data, instead the whole training data set is stored in one text file. So if I get this output:

2019-08-05 11:10:56,343 | end of split   1 /  1 | epoch  53 | time: 6588.93s | valid loss  0.99 | valid ppl     2.70 | learning rate 20.0000

normally a model will be saved?

**Comment by NDTanja** [comment]
Thank you for your quick response!

**Comment by alanakbik** [comment]
Great! Will close but feel free to reopen if you have more questions!
